tests/test_cancel_token.py

```diff
@@ -136,14 +136,15 @@ def test_token_chain_trigger_last():
         # (https://docs.python.org/3/library/asyncio-task.html#asyncio.Task.cancel), so we need
         # the sleep below before we check that the task is actually cancelled.
         await asyncio.wait_for(future, timeout=0.01)
-    await asyncio.sleep(0)
+    with pytest.raises(asyncio.CancelledError):
+        await asyncio.wait_for(future, timeout=0.01)
```

Comment:
I'm not clear which code change would switch this from red to green. Maybe a quick comment, or reference?